docs/broker.rst

```diff
@@ -88,6 +88,73 @@ its secret type updates to the secret object in the Kubernetes API will be autom
 a Pod's filesystem. Hpfeeds will spot those updates and process them immediately without needing a
 restart.
 
+Database authentication store
+--------------------------
+
+When starting the broker you can pass a database connection string. Auth requests are then checked against
+the selected Database in a table named auth_keys. 
+
+Supported Database drivers are:
+  - postrgesql
+  - mysql
+  - mysql compatable e.g. mariadb, aurora
+  - sqlite
+
+You may need to install the specific database driver for your selected database, more information can be 
+found at https://pypi.org/project/databases/
+
+.. code-block: bash
+
+  $ pip install databases[postgresql]
+  $ pip install databases[mysql]
+  $ pip install databases[sqlite]
+
+Using SQLite with this auth mechanism requires JSON support that can be found in SQLite version > 3.3 and Python3
+Previous versions of SQLite may be supported with the JSON1 SQLite extension.
```

Comment:
I feel like maybe this should mention the native dependency-less option is available?

Reply:
Agreed, will add that above. I was tempted just to ignore the fact that this also supported SQLite, but wasn't sure.

Reply:
I think it is better to use your new sqlite option in general - because its properly async, im not sure the built-in one actually is 😬